### PR TITLE
MAISTRA-1081 Allow users to configure the number of worker threads us…

### PR DESCRIPTION
**Pull request by luksa**
…ed by controllers

### CODE REVIEW

**Review by dgn — APPROVED** [review]
lgtm

File: pkg/controller/servicemesh/controlplane/controller.go [hunk 1]
```diff
@@ -272,12 +272,16 @@ func (r *ReconcileControlPlane) Reconcile(request reconcile.Request) (reconcile.
 }
 
 var reconcilers = map[string]*ControlPlaneReconciler{}
+var mu sync.Mutex
```
Comment (rcernich) [inline pkg/controller/servicemesh/controlplane/controller.go]: We only create one `ReconcileControlPlane`, but shouldn't this mutex be contained within that struct?

Comment (luksa) [reply pkg/controller/servicemesh/controlplane/controller.go]: The mutex is to protect the `reconcilers` map, that's why it's declared here. But you're right. The map itself should be contained in the struct.

Comment (rcernich) [reply pkg/controller/servicemesh/controlplane/controller.go]: Sorry I missed that.

**Comment by luksa** [comment]
@rcernich can I get a whoop whoop? (or at least an approval) :)

**Review by rcernich — APPROVED** [review]

**Pull request merged**